src/shared/components/download/Download.tsx

```diff
@@ -245,11 +254,34 @@ const Download: FC<DownloadProps> = ({
               value="false"
               checked={!compressed}
               onChange={handleCompressedChange}
+              disabled={redirectToIDMapping}
             />
             No
           </label>
         </fieldset>
       )}
+      {/* Peptide search download for matches exceeding the threshold */}
+      {redirectToIDMapping && (
+        <Message level="warning">
+          To download the peptide search results of more than{' '}
+          <LongNumber>{MAX_FACETS}</LongNumber> matches, we recommend using{' '}
```

Comment:
```suggestion
          <LongNumber>{MAX_FACETS}</LongNumber> matches, please use the{' '}
```
Just thinking that we're not really recommending but forcing them to use id mapping.